homework_03/18_stelian_todorichkov/test_circle.py

```diff
@@ -0,0 +1,38 @@
+from circle import Circle, Point, RealativePosition
+
+def test_same():
+    circle1 = Circle(Point(0.0, 0.0), 5.0)
+    circle2 = Circle(Point(0.0, 0.0), 5.0)
+    assert circle1.find_relative_position(circle2) == RealativePosition.SAME
+
+def test_intersecting():
+    circle1 = Circle(Point(0.0, 0.0), 2.0)
+    circle2 = Circle(Point(5.0, 0.0), 5.0)
+    assert circle1.find_relative_position(circle2) == RealativePosition.INTERSECTING
+
```

Comment:
For the sake of consistency, 1 more test case in the above tests wouldn't be extra.